textacy/tm/topic_model.py

```diff
@@ -103,9 +104,13 @@ class TopicModel(object):
         - http://scikit-learn.org/stable/modules/generated/sklearn.decomposition.TruncatedSVD.html
     """
 
+    _required_trained_model_attr = {"transform", "components_", "n_topics"}
```

Comment:
Scanning through the code, it looks like models also need a `.fit()` method and `n_components` attribute. Is that doable?

Reply:
agree with the `n_components`, but would be good to leave `.fit` method a bit optional, essentially it allows to initialize with a trained model, so we don't need to call the training method. But up to you I guess.